cernopendata_client/cli.py

```diff
@@ -213,8 +219,38 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
     default=False,
     help="Verify downloaded data file integrity.",
 )
+@click.option(
+    "--retry-limit",
+    "retry_limit",
+    default=DOWNLOAD_RETRY_LIMIT,
+    type=click.INT,
+    help="Number of retries when downloading a file. [deafult={}]".format(
+        DOWNLOAD_RETRY_LIMIT
+    ),
+)
+@click.option(
+    "--retry-sleep",
+    "retry_sleep",
+    default=DOWNLOAD_RETRY_SLEEP,
+    type=click.INT,
+    help="Sleep time before retrying to get a file from the server. [deafult={}]".format(
```

Comment:
Sleep time in seconds before retrying downloads.

(You can also modify `config.py`.)